### ISSUE TITLE
Standalone build is not usable

### ISSUE THREAD

**Issue by diegonc** [negomi/react-burger-menu#23]
I'm trying the standalone build located at the `dist` folder but an error is popping up claming that the 'react' module cannot be found.

BTW, I was able to use the standalone version of the v1.1.6 tag after applying [this](https://github.com/diegonc/react-burger-menu/commit/f5eede89b743bcf67e48ea0eda11ec07b06712f5) patch. Actually, that's the version I'm interested in because it supports React 0.13.[**]

Back to master... the javascript console shows the following exception:

```
Uncaught Error: Cannot find module 'react'
s                                 @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:1349
21../print-styles.js              @ react-burger-menu.js:1508
s                                 @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:1
24../components/print-style-sheet @ react-burger-menu.js:1549
s                                 @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:2853
41../BurgerIcon                   @ react-burger-menu.js:3037
s                                 @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:1
49.../menuFactory                 @ react-burger-menu.js:3304
s                                 @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:1
38../menus/bubble                 @ react-burger-menu.js:2732
s                                 @ react-burger-menu.js:1
e                                 @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:1
(anonymous function)              @ react-burger-menu.js:1
```

The code I'm using can be found [here](https://github.com/diegonc/diegonc.github.io/tree/master/tests/react-burger-menu/negomi-example-master/dist).

I'm not very familiar with package managers for the web and all the require stuff so I cannot make sense of what's going on.

My (little) understanding is that the browserify-shim should make the standalone version use the React symbol in the global scope instead of relying on require. At least that's how it worked for Snap.svg in v1.1.6.

Can you spot anything wrong?

----

[**] Actually, what I'm trying to do is to package this React component for ClojureScript, so a standalone version is what I need. And due to Om (the CLJS React wrapper) using React 0.13, I guess I'm stuck with version 1.1.6

**Comment by diegonc** [comment]
Ok, I think I found a possible fix:

```
$ git diff package.json
diff --git a/package.json b/package.json
index d8f0920..e949f46 100644
--- a/package.json
+++ b/package.json
@@ -45,7 +45,10 @@
     ]
   },
   "browserify-shim": {
-    "react": "global:React"
+    "react": "global:React",
+    "react-dom": "global:ReactDOM",
+    "radium": "global:Radium",
+    "snapsvg": "global:Snap"
   },
   "scripts": {
     "build": "gulp clean && NODE_ENV=production gulp build",
```
With that patch, things started to work after running `gulp build`.
Does it look sane?

**Comment by negomi** [comment]
Hey @diegonc, you're right, all of those need to be available as globals for the standalone build to work. I hadn't been keeping `browserify-shim` up to date. I'll release a patch for it today. Sorry about that!

**Comment by diegonc** [comment]
Great! Will you make a release for the older React too?

BTW, thanks a lot for the awesome menu! :-)

**Comment by negomi** [comment]
Ok, it should be fixed from v1.4.3. I can't change the old v1.1.6 release, but I've put the patched version on a [new branch](https://github.com/negomi/react-burger-menu/tree/1.1.6). The `dist/` files on that branch have been built with the updated `browserify-shim`, so should work standalone.

Let me know if you run into any more problems :)
